### PR TITLE
fix: remove deprecated jwks_url from config KV

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
fix: remove deprecated jwks_url from config KV

## Motivation and Context
remove unsupported Key

## How to test this PR?
Nothing special remove a deprecated Key.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

File: internal/config/identity/openid/jwt.go [hunk 1]
```diff
@@ -469,27 +469,29 @@ var (
...
 		return c, err
 	}
 
 	jwksURL := env.Get(EnvIamJwksURL, "") // Legacy
 	if jwksURL == "" {
-		jwksURL = env.Get(EnvIdentityOpenIDJWKSURL, kvs.Get(JwksURL))
+		jwksURL = env.Get(EnvIdentityOpenIDJWKSURL, "")
+	}
+	if jwksURL != "" {
+		return c, fmt.Errorf("legacy jwks_url is not supported anymore, please use %s", ConfigURL)
```
Comment (klauspost) [inline internal/config/identity/openid/jwt.go]: Do we need to fail? Can't we print a warning for a while?

Comment (harshavardhana) [reply internal/config/identity/openid/jwt.go]: @klauspost we can do that as well.

**Review by vadmeste — APPROVED** [review]
LGTM

**Review by klauspost — APPROVED** [review]
LGTM

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by Alevsk — APPROVED** [review]

**Pull request merged**